extensions/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~  Copyright 2019 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-config-parent-1.3</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+    </parent>
+    
+    <groupId>io.smallrye.ext</groupId>
+    <artifactId>smallrye-config-extensions</artifactId>
+    
+    <packaging>pom</packaging>
+    <name>SmallRye: MicroProfile Config Extensions</name>
+    
+    <modules>
+        <module>sources</module>
+        <module>utils</module>
+        <module>converters</module>
+        <module>example</module>
+    </modules>
+    
+    <properties>
+        <lombok.version>1.18.8</lombok.version>
+        <microProfile.version>2.2</microProfile.version>
+        <arquillian.version>1.4.1.Final</arquillian.version>
+    </properties>
+    
+    <dependencyManagement>
+        <dependencies>
+            <!-- MicroProfile -->
+            <dependency>
+                <groupId>org.eclipse.microprofile</groupId>
+                <artifactId>microprofile</artifactId>
+                <version>${microProfile.version}</version>
+                <type>pom</type>
+                <scope>provided</scope>
+            </dependency>
+            <!-- Lombok -->
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <!-- For testing -->
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile</groupId>
```

Comment:
Any reason the whole MP BOM is made available?

My concern is that would allow inadvertent use of non MP Config spec APIs in extensions to Config.

I would imagine it's only the example that needs this?

Reply:
Using other APIs from MP (JSON-P, CDI, JAX-RS). I can change to add those explicitly ?

Reply:
Changed to use the underlying apis and removed the MicroProfile depencency